lib/linux_kernel/processor.c

```diff
@@ -639,12 +648,12 @@ M0_INTERNAL int m0_processor_describe(m0_processor_nr_t id,
 
 	return processor_x86_info_get(id, pd);
 }
-
+#endif
 M0_INTERNAL m0_processor_nr_t m0_processor_id_get(void)
 {
 	return smp_processor_id();
 }
-
+#endif
```

Comment:
Add `/* CONFIG_X86_64 || CONFIG_AARCH64 */`

Reply:
Yes. Because the #ifdef is far away above.
When I check this line code, I scroll back and see it.